assets/src/js/Components/ArchivePicker/blockUpdateFunctions.js

```diff
@@ -0,0 +1,42 @@
+export const updateImageblockAttributes = (image, currentBlock) => {
+  const div = document.createElement('div');
+
+  if (image.id) {
+    // Check if the current block has content already, then just replace it.
+    if (currentBlock.originalContent) {
+      div.innerHTML = currentBlock.originalContent;
+      div.querySelector('img').setAttribute('class', `wp-image-${image.id}`);
+      div.querySelector('img').src = image.source_url;
+      if (div.querySelector('figcaption')) {
+        div.querySelector('figcaption').textContent = image.caption.raw;
+      }
+    } else {
+      // If there is no content then create block content.
+      const blockContent = `
+        <figure class="wp-block-image size-large">
+          <img src=${image.source_url} alt=${image.alt_text} class="wp-image-${image.id}"/>
+          <figcaption class="wp-element-caption">
+            ${image.caption.raw}
+          </figcaption>
+        </figure>`;
+      div.innerHTML = blockContent;
```

Comment:
What if you define it directly instead of create a `const`? 

```javascript
 div.innerHTML = `
        <figure class="wp-block-image size-large">
          <img src=${image.source_url} alt=${image.alt_text} class="wp-image-${image.id}"/>
          <figcaption class="wp-element-caption">
            ${image.caption.raw}
          </figcaption>
        </figure>`;
```